docs/pages/guides/CssOverrides.example.tsx

```diff
@@ -1,9 +1,8 @@
-// @ts-nocheck
 import React, { useState } from 'react';
 import lightBlue from '@material-ui/core/colors/lightBlue';
 import { TextField } from '@material-ui/core';
-import { DatePicker } from '@material-ui/pickers';
 import { createMuiTheme, ThemeProvider } from '@material-ui/core';
+import { DatePicker, MaterialUiPickersDate } from '@material-ui/pickers';
```

Comment:
I believe we encourage the following in the mono repository cc @eps1lon. Would it work? 

```suggestion
import { DatePickerProps } from '@material-ui/pickers';

DatePickerProps['value']
```

Reply:
Yes it will